src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -111,11 +128,24 @@ public static synchronized void initClient() {
                         TransactionWrapper.getSqlFromConfig(ApplicationConfigConstants.ES_USERNAME),
                         TransactionWrapper.getSqlFromConfig(ApplicationConfigConstants.ES_PASSWORD));
             } catch (MalformedURLException e) {
-                throw new RuntimeException("Error while initialize ES client", e);
+                throw new RuntimeException("Error while initializing ES client", e);
             }
         }
     }
 
+    public static void fetchFieldMappings() {
```

Comment:
I think having this method as `private` would be better, since it is used only in this class and I believe outside of this class does not need to know about it.

Reply:
correcting it